server/asset/dcr/dcr.go

```diff
@@ -127,7 +138,7 @@ func NewBackend(configPath string, logger dex.Logger, network dex.Network) (*Bac
 
 	// Ensure the network of the connected node is correct for the expected
 	// dex.Network.
-	net, err := dcr.client.GetCurrentNet()
+	net, err := dcr.client.GetCurrentNet(context.TODO())
```

Comment:
How about a `context.WithTimeout(context.Background(), 5 * time.Second)` to limit how long the constructor can hang.

Reply:
With the change from Runner => Connector just pushed, no more RPCs are in the constructor.